modules/gdscript/gdscript_editor.cpp

```diff
@@ -68,14 +68,21 @@ Ref<Script> GDScriptLanguage::make_template(const String &p_template, const Stri
 	if (!EDITOR_GET("text_editor/completion/add_type_hints")) {
 		processed_template = processed_template.replace(": int", "")
 									 .replace(": String", "")
+									 .replace(": Array[String]", "")
 									 .replace(": float", "")
 									 .replace(":=", "=")
+									 .replace(" -> String", "")
+									 .replace(" -> int", "")
 									 .replace(" -> void", "");
```

Comment:
This is not going to scale if we have to consider every type that an user might want to add to a custom template.

Reply:
Indeed, but that's beyond the point of this PR.